cli/src/semgrep/commands/scan.py

```diff
@@ -540,6 +540,13 @@ def convert(
         flag_value=EngineType.OSS,
         help="Run using only OSS features, even if the Semgrep Pro toggle is on.",
     ),
+    optgroup.option(
+        "--secrets",
+        "run_secrets_flag",
+        is_flag=True,
+        hidden=True,
```

Comment:
why this is an hidden flag? Anyway it does not matter because now osemgrep is the one used to display the semgrep scan --help.